dataproc_jupyter_plugin/handlers.py

```diff
@@ -134,7 +134,7 @@ def get(self):
                 cached_credentials = get_cached_credentials(self.log)
                 self.finish(json.dumps(cached_credentials))
             else:
-                t1 = threading.Thread(target=get_cached_credentials, args=())
+                t1 = threading.Thread(target=get_cached_credentials, args=([self.log]))
```

Comment:
The parentheses around `[self.log]` are unnecessary here and I think should be removed (they are just enforcing order of operations, but there's no operations being performed).

Note that this is very different from the original version where the empty parentheses defined an empty tuple. In this case, since there is exactly one expression inside the parentheses it is *not* a tuple.

Reply:
n/m, had to revert back to [self.log]

https://docs.python.org/3/library/threading.html#threading.Thread

Looks like args expects an iterable which the tuple provides.